Jenkinsfile

```
@@ -67,11 +67,10 @@ pipeline {
       }
     }
 
-    stage('Start File server for Integration Tests'){
+    stage('Start File-Server for Performance and Integration Testing'){
       steps{
         script{
-            sh 'scp src/test/resources/iudx-file-server-api.Release-v5.0.0.postman_collection.json jenkins@jenkins-master:/var/lib/jenkins/iudx/fs/Newman/'
-            sh 'docker compose -f docker-compose.test.yml up -d integTest'
+            sh 'docker compose -f docker-compose.test.yml up -d perfTest'
```

The service name defined in the compose is integTest not perfTest
```suggestion
            sh 'docker compose -f docker-compose.test.yml up -d integTest'
```